docs/examples/dialog/destroy-on-close.vue

```diff
@@ -10,19 +10,19 @@
     destroy-on-close
     center
   >
-    <span
-      >Notice: before dialog gets opened for the first time this node and the
-      one bellow will not be rendered</span
-    >
+    <span>
+      Notice: before dialog gets opened for the first time this node and the
+      one bellow will not be rendered
```

Comment:
You need to correct the format error here. try use the `pnpm lint:fix`

Reply:
OK, I have fixed this problem now, thank you